source/_current_clients.html.erb

```diff
@@ -73,7 +76,8 @@
         Greater Central Catholic High School (GCCHS)
       </div>
       <div class="collapsible-body">
-        <h5>Computer Science Teacher (Part-Time)</h5>
+        <h5><a href="http://www.gcchs.org/">Greater Central Catholic High School</a></h5>
```

Comment:
lets make all these links target blank so we will not see dropoffs on google analytics